src/FormattedMessage.tsx

```diff
@@ -1,14 +1,19 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { TextContext } from './TextProvider';
 
-const FormattedMessage = (props) => (
+interface FormattedMessageProps {
+  id: string;
+  values?: {};
```

Comment:
We can make `values` even stricter by allowing only `strings` as the key/value.

Reply:
yes, changed it to `values?: { [key: string]: string };`